elementary/utils/log.py

```diff
@@ -25,7 +28,30 @@ def format(self, record):
 
 
 FORMATTER = ColoredFormatter()
-LOG_FILE = "edr.log"
+LOG_FILE = os.getcwd() + "/edr_files/edr.log"
+
+
+class CustomerFileHandler(logging.FileHandler):
+    def emit(self, record):
+        file_path = os.getcwd() + "/edr_files"
+        try:
+            target_path_param = "target_path"
+            target_path_flag = "--target-path"
+            ctx = click.get_current_context()
+            ctx_args = ctx.args
+            if target_path_flag in ctx_args:
+                file_path = ctx_args[ctx_args.index(target_path_flag) + 1]
+            elif target_path_param in ctx.params:
+                file_path = ctx.params[target_path_param]
+        except Exception:
+            pass
+        finally:
+            os.makedirs(os.path.abspath(os.path.abspath(file_path)), exist_ok=True)
```

Comment:
we don't need to double `abspath` , right? 
```suggestion
            os.makedirs(os.path.abspath(file_path), exist_ok=True)
```